devlib/utils/misc.py

```diff
@@ -32,6 +32,7 @@
 from operator import itemgetter
 from itertools import groupby
 from functools import partial
+import wrapt
```

Comment:
Please add a space between stdlib imports and third party ones.

Also, this is a new dependency. Please add it to setup.py